akka-http-core/src/main/mima-filters/10.1.11.backwards.excludes/1354-add-HttpCookie-SameSite-attribute.backwards.excludes

```diff
@@ -0,0 +1,5 @@
+# Add SameSite attribute to HttpCookie
+
+ProblemFilters.exclude[ReversedMissingMethodProblem]("akka.http.javadsl.model.headers.HttpCookie.getSameSite")
```

Comment:
@jrudolph these are the only places where bin-compat is broken now.

Reply:
Right. So the 'excuse' here is that we don't expect users to extend those classes, even though we didn't explicitly mark them `@DoNotInherit`. That seems acceptable to me, but perhaps we should add those `@DoNotInherit` marks now?